_chapters/js/style.md

```diff
@@ -1,6 +1,45 @@
 ---
-title: Style
+title: Style / Linting
 slug: js-style
 ---
-We currently recommend adhering to the [AirBnB](https://github.com/airbnb/javascript) JavaScript style guide. 
-Your team may, of course, choose a different style for your project if you all agree that you have reason to do so.
+We recommend adhering to the [Airbnb JavaScript style guide](https://github.com/airbnb/javascript).
+
+Maintaining stylistic consistency across 18F's code helps lower the barrier to jumping in and helping
+with or reviewing other projects because we'll all be familiar with reading and working with code
+that looks similar. Having consistent rules for styling also removes generally non-productive
+discussions (aka bikeshedding) around personal code-formatting preferences.
+
+[`eslint`](http://eslint.org/) is our preferred tool for analyzing and flagging (aka "linting")
+JavaScript that is out of line with a set of stylistic rules. There are plugins to integrate
+eslint with nearly every code editor and build system, as described at [http://eslint.org/docs/user-guide/integrations](http://eslint.org/docs/user-guide/integrations).
+Using an eslint plugin with your editor makes it easier to quickly see non-conforming lines of
+code on the spot, as shown below:
+
+![Atom eslint plugin screenshot]({{ site.url }}/images/atom-eslint-example.png)
+
+Airbnb provides `npm` packages of `eslint` rules that implement their style guide:
+
+- [eslint-config-airbnb](https://www.npmjs.com/package/eslint-config-airbnb) for projects using ES2015 (and up) and React
```

Comment:
This is just stylistic, but I think it's easier to parse these if the tooling is listed first, such as:
```
- For projects using both ES2015 (or above) and React: link
- For projects using ES2015 (or above) without React: link
- For projects using ES5 and below: link
```

Also, should we refer to `ES6` instead of `ES2015` since we refer to `ES5`?